packages/webextension/index.js

```diff
@@ -0,0 +1,58 @@
+const web = require('@neutrinojs/web');
+const webExt = require('web-ext').default;
+const { join } = require('path');
+
+const MODULES = join(__dirname, 'node_modules');
+
+module.exports = (neutrino, options = {}) => {
+  neutrino.use(web, options);
+
+  if (neutrino.options.debug) {
+    webExt.util.logger.consoleStream.makeVerbose();
+  }
+
+  neutrino.config
+    .output
+      .delete('chunkFilename')
+      .hotUpdateChunkFilename('hot/hot-update.js')
+      .hotUpdateMainFilename('hot/hot-update.json')
+      .end()
+    .devServer
+      .clear()
+      .end()
+    .module
+      .rules
+        .delete('worker')
+        .end()
+      .end()
+    .plugins
+      .delete(options.pluginId || 'html')
+      .end()
+    .resolve
+      .modules
+        .add(MODULES)
+        .end()
+      .end()
+    .resolveLoader
+      .modules
+        .add(MODULES)
+        .end()
+      .end()
+    .when(neutrino.options.command === 'start', () => {
+      webExt.cmd.run({
```

Comment:
I don't know much about neutrino but is there a way to make some `run()` parameters configurable by the user? I can see a couple parameters [here](https://github.com/mozilla/web-ext/blob/1b22d60aefa31e2869e15d18ec52b39afa0de405/src/cmd/run.js#L31) that the user may want to configure such as `firefox` (path to executable), `firefoxProfile`, `browserConsole`, `pref`...

Reply:
Yep. I made all the options configurable in my latest commit. That was a good point!